### DISCUSSION TITLE
Support Collapsible banner ads

### DISCUSSION THREAD

**Discussion by Moritz-Karrasch** [marcojak/MauiMTAdmob#78] [Ideas]
Collapsible banner ads are banner ads that are initially presented as a larger overlay, with a button to collapse them to the originally requested banner size. Collapsible banner ads are intended to improve performance of anchored ads that are otherwise a smaller size. This guide shows how to turn on collapsible banner ads for existing banner placements.

Can you please add the new Collapsible banner ads?

Thank you

**Comment by marcojak** [comment]
Hi,

Collapsible banners will be available in the next release.
They are already implemented and tested on Android, and they are
fully functional.
I've implemented them on iOS but I still have to test them and they might
not work as the original ask is quite old.
If that's the case, I'll start to release the plugin and then work on a
newer version of the main admin SDK for iOS.

I hope to release the new version next week.

Best regards,
Marco

**Comment by Moritz-Karrasch** [comment]
Hi,

thank you very much.

Greetings 
Moritz
